CMSgov/onemac
# OY2-11469: give System Admin a State column in User Management

Story: https://qmacbis.atlassian.net/browse/OY2-11469
Endpoint: https://d1fxgbw8lbvckm.cloudfront.net/

### Details

Add "State" column in User Management table for CMS System Admin.

### Changes

- Just that one.

### Implementation Notes

N/A

### Test Plan

1. Log in as `systemadmintest`. Go to User Management.
2. Verify that the table now has a "State" column. For state users, this should indicate the state they have access to. For CMS/Helpdesk users, this should say "N/A".
